### ISSUE TITLE
Wiki links are currently broken

### ISSUE THREAD

**Issue by marcells** [Capgemini/Cauldron#58]
Switch to the Wiki on GitHub and try several links in the TOC. Most of them are broken,

**Comment by reflection-emit** [comment]
Oh... Its maybe because I changed the Sandcastle documentation file names from GUID to actual member names. Thanks for the hint. I will look into that ASAP.

________________________________

Firma: Capgemini Deutschland GmbH
Aufsichtsratsvorsitzender: Antonio Schnieder • Geschäftsführer: Dr. Michael Schulte (Sprecher) • Jost Förster • Dr. Peter Lempp • Dr. Volkmar Varnhagen

Amtsgericht Berlin-Charlottenburg, HRB 98814
This message contains information that may be privileged or confidential and is the property of the Capgemini Group. It is intended only for the person to whom it is addressed. If you are not the intended recipient, you are not authorized to read, print, retain, copy, disseminate, distribute, or use this message or any part thereof. If you receive this message in error, please notify the sender immediately and delete all copies of this message.

**Comment by reflection-emit** [comment]
Nope... Actually I started to write the documentation, but haven't finished it yet.

In short.. You can create a json file in your projects root folder and name it `*cauldron.fody.json`
Here is an example of how it could look like. 
```json
[
  {
    "is-active": true,
    "debug-only": false,
    "decorator": [
      {
        "type-name": "Program.WebService.LoggerAttribute", // The full name of the attribute to decorate
        "decorate-class": false,
        "decorate-methods": true,
        "decorate-properties": false,
        "target-class-filter": {
          "public": true, // Include all public class?
          "internal": true, // Include all internal class?
          "private": false, // Inlude all private class?
          "static": false, // Include all static class?
          "instanced": true, // Include all instanced class?
          "name": "\\w*Controller" // Regex - include all classes with that matches
        }
      },
      {
        "type-name": "Company.Common.Logging.PerformanceLoggerAttribute",
        "decorate-class": false,
        "decorate-methods": true,
        "decorate-properties": false,
        "target-class-filter": {
          "public": true,
          "internal": true,
          "private": false,
          "static": false,
          "instanced": true,
          "name": "\\w*Controller"
        }
      }
    ]
  }
]
```
There are more options though. I will be explaining them in the documentation. I hope I can finish it by the end of the month.

**Comment by krishna0209** [comment]
Thank you, this is really helpful!

**Comment by reflection-emit** [comment]
BTW... [Here ](https://github.com/Capgemini/Cauldron/blob/59a97529df15a4c61d5f39260283975b20726599/UnitTests/Win32_Fody_AssemblyWide_Validator_Tests/testconfig01-cauldron.fody.json)is a json with all the options which I use for the unit test as an example.
